pandas/core/reshape/pivot.py

```diff
@@ -424,16 +435,27 @@ def _convert_by(by):
 
 @Substitution("\ndata : DataFrame")
 @Appender(_shared_docs["pivot"], indents=1)
-def pivot(data: "DataFrame", index=None, columns=None, values=None) -> "DataFrame":
+def pivot(
+    data: "DataFrame",
+    index: Optional[Union[Label, Sequence[Label]]] = None,
+    columns: Optional[Union[Label, Sequence[Label]]] = None,
+    values: Optional[Union[Label, Sequence[Label]]] = None,
+) -> "DataFrame":
     if columns is None:
         raise TypeError("pivot() missing 1 required argument: 'columns'")
-    columns = columns if is_list_like(columns) else [columns]
+
+    if is_list_like(columns):
```

Comment:
see if you can use pandas.core.common.convert_to_list_like here

Reply:
yep, changed!

Reply:
can you change this one?

Reply:
emm? this one has been changed to using `convert_to_list_like`? @jreback

Reply:
whoops!! 

this is really my bad, somehow didn't commit the change here. sorry @jreback

Reply:
same